source/fundamentals/connection/socks.txt

```diff
@@ -0,0 +1,139 @@
+.. _node-connect-socks:
+
+===========================
+Enable SOCKS5 Proxy Support
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances with
+SOCKS5 proxy support. SOCKS5 is a standardized protocol for connecting
+to network services through a separate proxy server.
+
+.. tip::
+   
+   To learn more about SOCKS, see the Wikipedia entry on
+   :wikipedia:`SOCKS <SOCKS>`.
+
+Load the socks Module
```

Comment:
Suggestion:

I think using consistent and precise terminology for this would help the reader understand the task.

I think this should be "Install the socks Package" or "Install the socks Module". I think packages and modules are technically very similar, but package is more specific in that it's "packaged" using a package.json file.

See https://docs.npmjs.com/about-packages-and-modules#about-modules for more info.